app/assets/stylesheets/moderators.scss

```diff
@@ -213,36 +213,205 @@
     margin: 90px auto;
     border-radius: 12px;
   }
+
+  // Reaction Buttons START
+
+  .thumb-reactions {
+    display: flex;
+    justify-content: center;
+  }
+
   .reaction-button {
     position: relative;
-    width: 80px;
-    height: 80px;
     background: white;
     border-color: white;
-    margin: 30px 30px;
-    border-radius: 200px;
-    border: 0px;
-    padding: 12px 12px;
+    border: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 156px;
+    height: $su-10;
+    padding: $su-4 $su-6;
+    margin: $su-4 $su-2 $su-4 $su-4;
+    border-radius: $br-default;
+
+    &:last-child {
+      margin: $su-4 $su-4 $su-4 $su-2;
+    }
+
+    &:hover {
+      background: rgba(59, 73, 223, 0.05);
+    }
     &:active {
-      border: 0px;
+      border: 0;
+    }
+    &:hover .reaction-button-circle {
+      border-color: var(--accent-brand);
     }
   }
 
-  .reaction-button img {
+  .reaction-button-circle {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 0 auto;
+    width: $su-9;
+    height: $su-9;
+    border: solid 2px var(--base-20);
+    border-radius: 100%;
+    background: $white;
+  }
+
+  .reaction-button-circle img {
+    width: $su-7;
+  }
+
+  img.upvote {
+    -webkit-transform: rotate(180deg);
+    -moz-transform: rotate(180deg);
+    -ms-transform: rotate(180deg);
+    -o-transform: rotate(180deg);
+    transform: rotate(180deg);
+  }
+
+  .vote-text {
+    font-weight: $fw-bold;
+    font-size: $fs-base;
+    line-height: $lh-base;
+    margin-top: $su-2;
+    color: var(--base-90);
+  }
+
+  .reaction-checkmark {
+    opacity: 0;
     position: absolute;
-    top: 5px;
-    width: 70px;
-    left: 5px;
+    top: 63px;
+    left: 86px;
+    width: $su-5;
+    height: $su-5;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--accent-success);
+    border-radius: 100%;
+    border: 2px solid $white;
+    padding: 2px;
+    z-index: 3;
+    color: $white;
+  }
+
+  .vomit-reaction-section,
+  .how-this-works-section {
+    display: flex;
+    justify-content: center;
+    margin: 0 $su-4 $su-4 $su-4;
+  }
+
+  .reaction-vomit-button {
+    display: flex;
+    align-items: center;
+    border: none;
+    background: $white;
+    height: $su-7;
+    padding: $su-1 $su-2;
+    border-radius: $br-default;
+
+    img {
+      position: absolute;
+      left: 107px;
+      width: $su-6;
+      height: $su-6;
+    }
+
+    span {
+      margin-left: 35px;
```

Comment:
is it possible to use flex box here instead of `margin-left: 35px` to set the position?

Reply:
Here is a cleaner way to position the elements for Mark as Spam:

https://codepen.io/lisasy/pen/xxwgOEO